pants-plugins/src/python/internal_backend/utilities/register.py

```diff
@@ -73,88 +66,6 @@ def pants_setup_py(
     )
 
 
-def contrib_setup_py_context_aware_object_factory(parse_context) -> Callable:
```

Comment:
See https://github.com/pantsbuild/pants/pull/10177#pullrequestreview-438508210 ... I think that we should preserve contrib unless we are committing to an uber-deploy approach.